app/hotels/src/singleHotel/HotelDetailScreen.js

```diff
@@ -51,6 +68,20 @@ export class HotelDetailScreen extends React.Component<Props, State> {
     return !isPropsEqual || !isStateEqual;
   };
 
+  componentWillUnmount = () => {
+    if (Platform.OS === 'ios') {
+      StatusBar.setBarStyle('dark-content');
+    }
+  };
+
+  onScroll = (e: NativeEvent) => {
+    if (Platform.OS === 'ios' && e.nativeEvent.contentOffset.y > 180) {
+      StatusBar.setBarStyle('dark-content');
```

Comment:
I would also not make it transparent after we passed the picture, but I guess it's up to the designer. Imho, you cannot really read the status bar correctly once it's over some text.

Reply:
No, default is the default value. But sorry, there is another method `setHidden` which will hide it. Personally id dislike hiding it

Reply:
Yeah, I would not like it to be hidden either. So I'll approve it as it is.

Reply:
```
Apple's official word is: 

Think twice before hiding the status bar if your application is not a game or full-screen media-viewing application. Although these apps might permanently hide the status bar, you should understand the ramifications of this design decision. Permanently hiding the status bar means that users must quit your app to find out, for example, whether they need to recharge their device.
```